sale_order_lot_generator/product.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#  licence AGPL version 3 or later
+#  see licence in __openerp__.py or http://www.gnu.org/licenses/agpl-3.0.txt
+#  Copyright (C) 2014 Akretion (http://www.akretion.com).
+#  @author Adrien CHAUSSENDE <adrien.chaussende@akretion.com>
+#
+##############################################################################
+
+from openerp import models, fields
+from openerp.osv import orm, fields as oldfields
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    sale_prodlot_generation = fields.Boolean(
```

Comment:
a better name will be auto_generate_prodlot. Indeed in case of a shipping generated by a RMA we can use the same behaviour. It's not linked to the sale